unitTests/ui/DynamicFacet/DynamicFacetTest.ts

```diff
@@ -227,6 +227,16 @@ export function DynamicFacetTest() {
       expect($$(test.cmp.element).hasClass('coveo-dynamic-facet-collapsed')).toBe(true);
     });
 
+    it(`when enableCollapse is false & collapsedByDefault options is true
```

Comment:
Would be also nice to detect those kind of invalid configuration (options that contradict one another) during component initialization/constructor, and log warnings in the console.

Reply:
He already does with the function `verifyCollapsingConfiguration`